packages/atomizer/bin/atomizer

```diff
@@ -106,35 +102,10 @@ function parseFile (file, recursive, dir) {
 }
 
 function buildAtomicCss(additionalFiles, done) {
-    // Custom rulesets
```

Comment:
This was moved outside of this function. This function is called anytime a new file is added or removed from the watcher. Since the list of rules doesn't change based on the files parsed, it didnt't make sense to add this each time. It will get run once when the script is executed.